packages/postgres-database/src/simcore_postgres_database/migration/versions/2ca532b0831f_new_products_table.py

```diff
@@ -0,0 +1,105 @@
+"""new products table
+
+Revision ID: 2ca532b0831f
+Revises: 90c92dae8fc9
+Create Date: 2022-11-11 10:54:13.921120+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "2ca532b0831f"
+down_revision = "90c92dae8fc9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
In the interest of time, @odeimaiz and me decided that this had a very low return of investment.
I think in this case it is "affordable" since this is *read-only* data and inserted manually via adminer. In addition, it is version-controled separately in the repo indicated above.